server/lib/nl/fulfillment/base.py

```diff
@@ -188,7 +189,7 @@ def _maybe_switch_parent_type(
 def _add_charts_with_existence_check(state: PopulateState,
                                      places: List[Place]) -> bool:
   svs = state.uttr.svs
```

Comment:
Drop the logging (we don't really use it)?   Also is `svs` variable used?

Reply:
dropped! the `svs` variable was used later in line 251, but removed `svs` now and updated that line to just use `state.uttr.svs`